docs/quick-reference.adoc

```diff
@@ -1,2 +1,215 @@
 = Sylt Quick Reference
 :favicon: sylt.png
+
+Here you will find most things in Sylt in a concise format.
+
+.Variables
+[source, sylt]
+----
+constant :: 3.14
+variable := []
+two: int : 2
+text: str = "abc"
+text = "A short sentence."  // <1>
+----
+
+<1> A single `=` is used to set the value of a variable after declaration with
+`:=`.
+
+.If-else statements
+[source, sylt]
+----
+if 1 < 0 do
+    // ...
+else if "hello" == "" do
+    // ...
+else do
+    // ...
+end
+
+one :: 1 if true else 2
+two :: one + 1 if == 2 else -1 // <1>
+----
+
+<1> The left hand side is inserted right after `if`
+
+.Operators
+[source, sylt]
+----
+1 + 1
+2 - 2
+3 * 3
+10 / 2
+-4
+
+1 < 2
+2 > 1
+1 <= 1
+1 >= 1
+"a" == "a"
+
+[1, 2] <=> [1, 2]  // Assert equal
+
+(1 + 1) * 2
+
+true or false
+true and true
+not true
+----
+
+.Functions
+[source, sylt]
+----
+hello :: fn do
+    print("Hello world")
+end
+
+add :: fn x: int, y: int -> int do
+    ret x + y
+end
+
+two_squared :: fn n: int -> int do n * n end(2)  // <1>
+
+// Different ways of calling functions
+
+hello()
+print("Hello, world!")
+add(1, 2)
+
+hello'
+print' "Hello, world!"
+add' 1, 2
+
+"Hello, world!" -> print()
+1 -> add' 2
+
+[1, 2, 3] -> for_each' fn n: int do
+    print(n)
+end
+----
+
+<1> This creates a function which returns implicitly and calls it immediately.
```

Comment:
```suggestion
<1> This creates a function with an implicit return. The function is called it immediately.
```
The given phrasing can mean that the implicitly returned _value_ is called. I think this phrasing is less prone to misinterpretation.